src/scss/site/_dashboard.scss

```diff
@@ -0,0 +1,94 @@
+#explore {
```

Comment:
Is there a reason why these styles should live in a Sass partial instead of the `Dashboard.svelte` component? Or are these styles used elsewhere besides Tools?

Reply:
I can move the `#explore` styles inside `extreme-weather/index.svelte`. If I move these into `Dashboard.svelte` , each rule will need a `:global(...)` modifier. And also styles that are in `:global(...)` blocks cannot have any nesting like scss styles.

Reply:
Got it. One other thought @mukhtyar is if these colors correspond to the climate models, should we have them stored as Sass & CSS variables in the `_variables.scss` partial so they can be referenced elsewhere?

Reply:
Yes that's a good idea. I'll add them in.